doc/faq.rst

```diff
@@ -207,3 +207,15 @@ temporarily disable your antivirus software to find out if it is the cause for
 your performance problems. If you are certain that the antivirus software is
 the cause for this and you want to gain maximum performance, you have to add
 the restic binary to an exclusions list within the antivirus software.
+
+How do I choose a strong password?
+----------------------------------
+
+Length is the single most important component in password strength.  That doesn't
+mean that other components such as complexity and entopy (or randomness) are not
+important to consider.  A strong password includes Alphabetical, Numerical and
+Special characters. For example, ``nk3E9Rr26md6GGySyyWMrfakw8Jck4$&vVY6`` would
+be a very strong password, if not for being in this documentation.
+
+There are plenty of tools out there, such as OpenSSL or KeePass that can generate
```

Comment:
I guess you refer to something like `openssl rand -base64 40`? I actually prefer `pwgen` which is probably much more obvious to use.

Reply:
I do.  Mostly because it's on the overwhelming majority of systems, so it reduces the dependency list.  pwgen is not installed on any of my machines at the moment, installed with CentOS, Debian and Ubuntu.  I opted for a smaller dependency tree at the cost of command complexity.  This is a case of personal preference bleeding into docs.

I've added mention of pwgen in the statement on line 220.